pkg/credentialprovider/aws/aws_credentials_test.go

```diff
@@ -108,56 +108,3 @@ func TestEcrProvide(t *testing.T) {
 		}
 	}
 }
-
-func TestChinaEcrProvide(t *testing.T) {
```

Comment:
in what way was this unused?

Reply:
the TestEcrProvide func has tested,only the para "cn-foo-1" and "lala-land-1" deffierent,I don't know if I understand wrong,thanks